PSK.UII/ClientApp/src/components/Layout/NavMenu.js

```diff
@@ -21,12 +38,19 @@ export class NavMenu extends Component {
     });
   }
 
+  logout(){
+    this.props.logout();
+    removeCookie('AuthToken');
+    this.props.history.push('/');
+    get("login/logout");
```

Comment:
Logoutas turėtų būti su post'u. Get'us browseriai cacheina ir dėl to galim turėt problemų

Susijusi info, jei kam įdomu :D
https://twitter.com/rombulow/status/990684453734203392